cep-6.md

```diff
@@ -0,0 +1,215 @@
+
+The notification message will appear while running the following commands:
+
+- `create`
+- `env create`
+- `install`
+- `update`
+- `env update`
+- `search`
+
+The reasoning behind this decision is that the above commands all retrieve `repodata.json` from the configured
+channels. Simply adding another file request (which is very small and immediately cached) would not add much more
+overhead to the execution of these commands. Additionally, these are commands where our users already expect network
+traffic to occur, so the requirement for having an active internet connection is assumed.
+
+Here's an example of how a notification message may appear while running the `conda create` command:
+
+```
+$ conda create -n geopandas geopandas
+Collecting package metadata (repodata.json): done
+Solving environment: done
+
+## Package Plan ##
+
+  environment location: /Users/username/opt/conda_x86_64/envs/geopandas
+
+  added / updated specs:
+    - geopandas
+
+... (output truncated)
+
+Notice [info]:
+> Here is a message to the user
+> Here is a link they could click: https://example.com/link-name
+> To see the message again, run `conda alert`
+
+```
+
+### How else can our users access this message?
+
+Additionally, because our users may wish to see this message on demand, we will add a new sub-command called `alerts`. 
+The following are a couple examples to show exactly how it would function:
+
+**Basic usage:** grabs notifications for all current channels:
+
+```
+$ conda alerts
+
+Channel: defaults
+
+Notice [info]:
+This is a test message. It is not very long, and could have a link to a longer post:
+https://example.com/short-link
+
+Channel: conda-forge
+
+Notice [info]:
+Here is another message. It could have info about the latest happenings or blog posts from conda-forge:
+https://conda-forge.org/
+```
+
+**Show a single channel:** grabs notifications for a single channel:
+
+```
+$ conda alerts -c default
+
+Notice [info]:
+This is a test message. It is not very long, and could have a link to a longer post:
+https://example.com/short-link
+```
+
+### What file format will this message have, and what will it contain?
+
+The notification message will be in the JSON file format. This will allow us to not only store the message itself but 
+also metadata about the message, including information about how often the client should display the message (more on 
+this in the next section).
+
+Here's an example of the `notifications.json` file which will be stored in the root of the channel directory structure.
+
+```json
+{
+  "notifications": [
+    {
+      "id": "1cd1d8e5-d96c-42d1-9c29-e8120ad80823",
+      "message": "Here is an example message that will be displayed to users",
+      "level": "info",
+      "created_at": "2022-04-26T11:50:34+00:00",
+      "expiry": 604800
+    }
+  ]
+}
+```
+
+Detailed overview of the JSON fields:
+
+- **notifications** [Array] holds zero or more notifications that will be displayed to the client.
```

Comment:
I disagree that `alerts` is appropriate as a name since it predefines the messages as something to react to, something alarming, while the use cases above are by design left to the channel authors to decide whether they should be alarming or not. I'd suggest to use the neutral `notices`.